supabase/functions/upload-to-storage/index.ts

```diff
@@ -156,142 +171,154 @@ serve(async function handler(req: Request) {
     doc.setTextColor(0, 0, 0);
     doc.setFont("Nunito", "normal");
 
-    autoTable(doc, {
-      ...tableStyles,
-      columnStyles: {},
-      head: [["Summary of Qualification"]],
-      body: [[employee.summary || ""]],
-      margin: { ...tableStyles.margin, top: 40 },
-    });
+    if (employee.summary) {
+      autoTable(doc, {
+        ...tableStyles,
+        columnStyles: {},
+        head: [["Summary of Qualification"]],
+        body: [[employee.summary || ""]],
+        margin: { ...tableStyles.margin, top: 40 },
+      });
+    }
 
     const skillsBody: Array<[string, string[], string[]]> = [];
 
-    Object.entries(skills).map(([groupName, skills]) => {
-      const skillList1: Array<string> = [];
-      const skillList2: Array<string> = [];
-
-      // separate skills in two columns
-      for (let i = 0; i < skills.length; i++) {
-        if (i % 2 === 0) {
-          skillList1.push(skills[i]);
-        } else {
-          skillList2.push(skills[i]);
+    if (skills) {
```

Comment:
Skills, projects and summary should always be present, but I just wanted to be sure the application would not fail in case it somehow happens.

Reply:
We also need to check if the heading is added even though no skills are added, and the same for the other tables that can be omitted. I got this strange behaviour on my local, not sure why.